src/components/rate/rate.service.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const rateModel = require('./rate.model');
+const mongoose = require('mongoose');
+
+/**
+ * Get all rates for a mediaItem and mediaType
+ * @param {String} mediaType 'movie' or 'series'
+ * @param {number} mediaId mediaId
+ * @param {String} accountId accountId
+ * @throws {Unauthorized} When credentials are wrong or not provided
+ * @throws {InternalServerError} When there's an unhandled error.
+ */
+module.exports.getAllRates = async (mediaType, mediaId, accountId) => {
+    try {
+        const rates = await rateModel.find({ mediaType: mediaType }).find({ mediaId: mediaId });
+
+        // adds ratedByMe=true if the rate was made by the logged user. else, adds ratedByMe=false
+        // const processedRates = rates.map(r => ({ ...r, ratedByMe: (r.accountId.toString() === accountId) }));
+
+        // console.log(processedRates);
+        return rates;
+    }
+    catch (error) {
+        throw new error.InternalServerError('Unexpected error updating the list');
```

Comment:
The error message is incorrect

Reply:
Oops, i guess it just happened when i pressed ctrl+v 😛